src/components/messenger/lib/utils.tsx

```diff
@@ -34,3 +35,17 @@ export const highlightFilter = (text, filter) => {
 
   return text;
 };
+
+export function getUserHandle(primaryZID: string, wallets: Wallet[]) {
```

Comment:
At some point it'd be good to change this to `getUserSubHandle` since we use their name as their handle.